step-1-getting-started.md

```diff
@@ -2,15 +2,23 @@
 
 ## Check Java
 You'll need Java 17 or newer for this workshop. 
-Testcontainers libraries are compatible with Java 8+, but this workshop uses a Spring Boot 3.1 application which requires Java 17 or newer.
+Testcontainers libraries are compatible with Java 8+, but this workshop uses a Spring Boot 3.x application which requires Java 17 or newer.
+
+## Install Testcontainers Desktop
+Testcontainers Desktop is a free tool that can help you during the local development and debugging of your tests.
+You can download the app from [https://testcontainers.com/desktop/](https://testcontainers.com/desktop/).
 
 ## Check Docker
 
-Make sure you have a Docker environment available on your machine. 
+To use Testcontainers, you need to have a supported Docker environment.
+If you have the Testcontainers Desktop application installed, it will show you the available container runtimes on your machine.
 
+* You can choose any of the installed container runtimes like **Docker Desktop**, **OrbStack**, **Rancher Desktop**, etc.
```

Comment:
@sivaprasadreddy Do you know if Rancher Desktop works throughout the workshop without any issues?

Reply:
Tried running workshop with TCD + Rancher Desktop and it worked well for me.